convert2rhel/utils.py

```diff
@@ -471,10 +471,17 @@ def download_pkg(
 
     output, ret_code = run_cmd_in_pty(cmd, print_output=False)
     if ret_code != 0:
-        loggerinst.warning(
-            "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
-        )
+        loggerinst.warning("you are using an env that overides an inhibitor and 
+            you understand that if you continue you might lose the rollback."
+            \n"Output from the yumdownloader call:\n%s" % (pkg, output)
+            )
+            cont = user_prompt("\nContinue with the system conversion? [y/n]: ")
+ 
+            if user_prompt == "y":
+                break
+
+            if user_prompt == "n":
+                sys.exit("User canceled the conversion\n")
```

Comment:
Here you could just replace this part of the code with an actual call to the function [ask_to_continue](https://github.com/oamg/convert2rhel/blob/874e25a4fad8e40a31cec6b0006414f4b419394b/convert2rhel/utils.py#L232), since it does exactly what you want to do and you can just reuse the function.

Reply:
the new code doesn't require any immediate action from the user so a prompts isn't needed.